collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -149,9 +149,13 @@ private static void rotateImageIfNeeded(String imagePath) {
     }
 
     private static void rotateBitmap(Bitmap image, int degrees, String imagePath) {
-        Matrix matrix = new Matrix();
-        matrix.postRotate(degrees);
-        image = Bitmap.createBitmap(image, 0, 0, image.getWidth(), image.getHeight(), matrix, true);
+        try {
+            Matrix matrix = new Matrix();
+            matrix.postRotate(degrees);
+            //image = Bitmap.createBitmap(image, 0, 0, image.getWidth(), image.getHeight(), matrix, true);
```

Comment:
I don't think this should be commented out?

Reply:
Ohhhh.. thanks for catching. Fixed.

Reply:
You just wanted to make sure I was paying attention, right? ;)

👍 Good fixes all around.